docs/introduction-to-smart-contracts.rst

```diff
@@ -133,7 +133,7 @@ too far, though, as it is neither possible to obtain a list of all keys of
 a mapping, nor a list of all values. So either keep in mind (or
 better, keep a list or use a more advanced data type) what you
 added to the mapping or use it in a context where this is not needed,
-like this one. The getter function created by the ``public`` keyword
+like this one. The :ref:`getter-functions` created by the ``public`` keyword
```

Comment:
I get what you mean, it should still say getter function, but I think the edit would make it say getter functions.